magpie-aws/src/test/java/io/openraven/magpie/plugins/aws/discovery/services/IAMRoleDiscoveryIT.java

```diff
@@ -67,19 +67,19 @@ public void testRoleDiscovery() {
   private void assertRole(MagpieEnvelope envelope) {
     var contents = envelope.getContents();
     assertNotNull(contents.get("documentId"));
-    assertTrue(contents.get("arn").asText().contains(ROLE_NAME));
+    assertTrue(contents.get("assetId").asText().contains(ROLE_NAME));
     assertTrue(contents.get("resourceName").asText().contains(ROLE_NAME));
     assertNotNull(contents.get("resourceId").asText());
     assertEquals("AWS::IAM::Role", contents.get("resourceType").asText());
-    assertEquals(ACCOUNT, contents.get("awsAccountId").asText());
-    assertEquals(BASE_REGION.toString(), contents.get("awsRegion").asText());
+    assertEquals(ACCOUNT, contents.get("accountId").asText());
+    assertEquals(BASE_REGION.toString(), contents.get("region").asText());
   }
 
   private void assertConfiguration(MagpieEnvelope envelope) {
     var configuration = envelope.getContents().get("configuration");
     assertEquals("/", configuration.get("path").asText());
     assertNotNull(configuration.get("roleId"));
-    assertTrue(configuration.get("arn").asText().contains(ROLE_NAME));
+    assertTrue(configuration.get("assetId").asText().contains(ROLE_NAME));
```

Comment:
this should remain `arn`. This is AWS specific configuration data received from AWS client